tests/backup/backup_psa_test.go

```diff
@@ -898,6 +899,10 @@ var _ = Describe("{RestoreFromHigherPrivilegedNamespaceToLower}", Label(TestCase
 	})
 	JustAfterEach(func() {
 		defer EndPxBackupTorpedoTest(scheduledAppContexts)
+
+		//Resetting the pipeline app list
+		Inst().AppList = pipelineAppList
```

Comment:
Let's put it in defer so that it doesn't break the next testcases:
```
		defer func() {

		}()
```

Reply:
done